builtin/providers/aws/resource_aws_autoscaling_group_test.go

```diff
@@ -262,7 +264,7 @@ resource "aws_launch_configuration" "foobar" {
 
 resource "aws_autoscaling_group" "bar" {
   availability_zones = ["us-west-2a"]
-  name = "foobar3-terraform-test"
+  name = "foobar3-terraform-test-%d"
```

Comment:
Not sure I follow.. this ASG is attached to `${aws_elb.bar.name}` and `${aws_launch_configuration.foobar.name}`, neither of which reference this ASG. Am I missing a connection between them?